### ISSUE TITLE
Invalid machine error (Hyper V-related)

### ISSUE THREAD

**Issue by timotei** [docker/machine#2945]
Hello,

I'm trying to create a new machine, but I get the following error:

```
Docker Machine Version:  0.5.6, build 61388e9
Found binary path at C:\Program Files\Docker Toolbox\docker-machine.exe
Launching plugin server for driver virtualbox
Plugin server listening at address 127.0.0.1:60507
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(flag-lookup) Calling .GetMachineName
(flag-lookup) Calling .DriverName
(flag-lookup) Calling .GetCreateFlags
Found binary path at C:\Program Files\Docker Toolbox\docker-machine.exe
Launching plugin server for driver virtualbox
Plugin server listening at address 127.0.0.1:60511
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(default) Calling .GetMachineName
(default) Calling .DriverName
(default) Calling .GetCreateFlags
(default) Calling .SetConfigFromFlags
Running pre-create checks...
(default) Calling .PreCreateCheck
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe --version
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | 5.0.14r105127
(default) DBG | }
Error creating machine: Error with pre-create check: This computer has Hyper-V installed. VirtualBox refuses to boot a 64bits VM when Hyper-V is installed. See https://www.virtualbox.org/ticket/12350
```

However, this is incorrect when booting with `hypervisorlaunchtype = off`, which is my case. I run just fine all kind of 64 bit VMs.

**Comment by jeanlaurent** [comment]
thanks @timotei, can you try to create the machine with the `--virtualbox-no-vtx-check`, to see if the machine runs ?
If this is the case, we will have to enhance our vtx check with the `hypervisorlaunchtype = off`.

**Comment by timotei** [comment]
Yup, with `--virtualbox-no-vtx-check` it worked! Thanks.

**Comment by dgageot** [comment]
@timotei Could you help use find out how to better detect if hyper-v is installed but not activated.
We currently rely on `vmms.exe` being in the path. It's obviously too aggressive.
I'll try to find a better way but we could use your help.

**Comment by timotei** [comment]
Do you want something that doesn't require elevated permissions?

A first idea would be to parse the ouput of: `bcdedit /get {current}` and check what value `hypervisorlaunchtype` has (auto = enabled, off or key missing  = disabled)

**Comment by ericis** [comment]
This just happened to me on Docker Toolbox 1.10.3 and virtualbox 5.0.16 on Win10 Enterprise 64-bit (latest updates).  I also had `hypervisorlaunchtype` disabled with `off` after booting from a different boot entry (`bcdedit`).

It also gave me the error: `Error creating machine: Error with pre-create check: This computer has Hyper-V installed. VirtualBox refuses to boot a 64bits VM when Hyper-V is installed.`

I had to exit Toolbox, run `cmd` with Administrative privileges, and manually run `docker-machine create --driver=virtualbox --virtualbox-no-vtx-check default`.

After running, I was able to start Docker Toolbox.
